src/components/JobConfiguration.vue

```diff
@@ -70,13 +70,34 @@
       <ion-button expand="block" @click="saveChanges()">{{ $t("Save changes") }}</ion-button>
     </div>
   </section>
+  <div class="job-actions">
+    <ion-item slot="start" @click="viewJobHistory(currentJob)" button>
+      <ion-icon slot="start" :icon="timeOutline" />
+      {{ $t("History") }}
+    </ion-item>
+    <ion-item slot="end" @click="runNow(title, currentJob)" button>
+      <ion-icon slot="start" :icon="flashOutline" />
+      {{ $t("Run now") }}
+    </ion-item>
+    <ion-item slot="start" @click="copyJobInformation(currentJob)" button>
+      <ion-icon slot="start"  :icon="copyOutline" />
+      {{ $t("Copy details") }}
+    </ion-item>
+    <ion-item slot="end" @click="updatePinnedJobs(currentJob?.systemJobEnumId)" button>
+      <ion-icon slot="start" :icon="pinOutline" />
+      {{ $t("Pin job") }}
+      <ion-checkbox :checked="getPinnedJobs.includes(currentJob.systemJobEnumId)" slot="end" />
```

Comment:
I am seeing slot in different places. Why not at the same place everytime?

Reply:
Updated sir